graphitepager/worker.py

```diff
@@ -113,11 +117,14 @@ def run(args):
                     target,
                     from_=alert.get('from'),
                     until_=alert.get('until'),
+                    http_connect_timeout_s_ = http_connect_timeout_s,
+                    http_read_timeout_s_    = http_read_timeout_s,
                 )
-            except requests.exceptions.RequestException:
+            except requests.exceptions.RequestException as e:
                 if not alert.alert_data['allow_no_data']:
                     print "Error, {0}".format(alert.alert_data)
                     update_notifiers_missing(notifier_proxy, alert, config)
+                print "Exception: %s" % e
```

Comment:
Gets us a _lot_ more info on error, but be warned it does include secrets e.g.:
```
2018-08-06T18:59:09.691075+00:00 app[worker.1]: Error, {'until': '-2min', 'exclude': set([]), 'warning': 3, 'from': '-60min', 'critical': 4, 'target': 'sumSeries(postgres.pkey.*.unhappy)', 'name': 'PG Improper PRIMARY KEY Tables', 'allow_no_data': False, 'doc_url': None, 'units': 'short'}
2018-08-06T18:59:09.771649+00:00 app[worker.1]: Exception: HTTPSConnectionPool(host='www.hostedgraphite.com', port=443): Max retries exceeded with url: /f03b3fab/ELIDED_BY_JHW_GRAPHITE_API_KEY/graphite//render/?target=sumSeries(postgres.pkey.*.unhappy)&rawData=true&from=-60min&until=-2min (Caused by ConnectTimeoutError(<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0x7f6304ea8bd0>, 'Connection to www.hostedgraphite.com timed out. (connect timeout=0.01)'))
```
Note `ELIDED_BY_JHW_GRAPHITE_API_KEY`, which I manually edited out from the GUID which it was in the original log message.